dashboard/db/migrate/20170504164217_remove_name_from_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNameFromPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :plc_courses, :name
```

Comment:
If we wanted to be even more conservative, we could rename the column, make sure nothing breaks (this way we can rollback if something does), and then at a future point, remove it.

That said, even if something were to go horribly awry, we should be able to recover this data from `::Course.name`